CMakeLists.txt

```diff
@@ -135,6 +135,8 @@ else ()
 endif ()
 option (jemalloc "Enables jemalloc for heap profiling" OFF)
 option (werr "treat warnings as errors" OFF)
+option (local_protobuf
```

Comment:
force_local_protobuf? local_protobuf seems like something discovered during config vs. an option.

I like options that have a verb in them. "FORCE_USE_LOCAL_PROTOBUF" "USE_JEMALLOC" "USE_WERR" etc.

Reply:
I mostly prefer shorter option names just to keep configure commands from getting too crazy, but I see your point. I'm going to keep this as is for now, but if we want to take a swipe at renaming our various configure options to make them more verb-oriented in the future, that would be fine with me.